plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -115,14 +113,7 @@ public class MachineLearningPlugin extends Plugin implements ActionPlugin {
     private ClusterService clusterService;
     private ThreadPool threadPool;
 
-    public static final Setting<Boolean> IS_ML_NODE_SETTING = Setting.boolSetting("node.ml", false, Setting.Property.NodeScope);
```

Comment:
curiosity question: what is the purpose of `IS_ML_NODE_SETTING` before and why we don't need it now?

Reply:
That part was from prototype when we try to support ML node. But actually it's not being used in our formal release as  OpenSearch core doesn't support ML role and we have no postpone that. Now OpenSearch supported ML role with dynamic role feature. We can add this back but we don't need this prototype/experiment code any more. Just move it to test part.